packages/discord.js/src/util/Util.js

```diff
@@ -65,6 +65,11 @@ function flatten(obj, ...props) {
  * @property {boolean} [spoiler=true] Whether to escape spoilers or not
  * @property {boolean} [codeBlockContent=true] Whether to escape text inside code blocks or not
  * @property {boolean} [inlineCodeContent=true] Whether to escape text inside inline code or not
+ * @property {boolean} [escape=true] Whether to escape escape characters or not
+ * @property {boolean} [heading=false] Whether to escape headings or not
+ * @property {boolean} [bulletedList=false] Whether to escape bulleted lists or not
+ * @property {boolean} [numberedList=false] Whether to escape numbered lists or not
+ * @property {boolean} [maskedLink=false] Whether to escape masked links or not
```

Comment:
Superfluous wording:

```suggestion
 * @property {boolean} [escape=true] Whether to escape escape characters
 * @property {boolean} [heading=false] Whether to escape headings
 * @property {boolean} [bulletedList=false] Whether to escape bulleted lists
 * @property {boolean} [numberedList=false] Whether to escape numbered lists
 * @property {boolean} [maskedLink=false] Whether to escape masked links
```

Reply:
This is for consistency with the above lines. Should I remove "or not" from those as well?

Reply:
Yeah, sure.